scripts/copy-files.js

```diff
@@ -42,7 +42,7 @@ async function createModulePackages({ from, to }) {
       ]);
 
       if (!typingsExist) {
-        throw new Error(`index.d.ts for ${directoryPackage} is missing`);
+        console.error(`index.d.ts for ${directoryPackage} is missing`);
```

Comment:
Seems like something that was used during debugging? As far as I know we shouldn't fail with zero exit-code because the package.json files do reference these.

You can add some stubs to satisfy the script:

```ts
declare const Pagination: any;
export default Pagination;
```

Reply:
Perfect, thanks.